src/main/webapp/manage-sessions.js

```diff
@@ -144,3 +144,56 @@ function cancelTutorSession(window, scheduledSession) {
         }
     });
 }
+
+/** Creates a calendar with the Charts API and renders it on the page  */
+function createCalendar() {
+    fetch('/confirmation', {method: 'GET'}).then(response => response.json()).then((scheduledSessions) => {
+        console.log(scheduledSessions);
+        // Don't create a calendar if there are no scheduled sessions
+        if (scheduledSessions === undefined || scheduledSessions.length == 0) {
+            return;
+        }
+
+        // There are available timeslots, display header and calendar
+        document.getElementById('calendar-header').style.display = 'block';
+        
+        const container = document.getElementById('calendar');
+        const chart = new google.visualization.Timeline(container);
+
+        const dataTable = new google.visualization.DataTable();
+        dataTable.addColumn({type: 'string', id: 'Date'});
+        dataTable.addColumn({type: 'string', id: 'Description'});
+        dataTable.addColumn({type: 'date', id: 'Start'});
+        dataTable.addColumn({type: 'date', id: 'End'});
+        
+        for (var session of scheduledSessions) {
```

Comment:
Does this show all upcoming sessions or just for a specific time?

Reply:
It shows all upcoming sessions